src/mame/rockwell/aim65.cpp

```diff
@@ -300,28 +299,111 @@ void aim65_state::aim65(machine_config &config)
 	SOFTWARE_LIST(config, "cart_list").set_original("aim65_cart");
 }
 
+void aim65_state::drac1(machine_config &config)
+{
+	aim65(config);
+
+	/* Expansion boards.
+	   Not currently emulated. */
+
+	/* 
+	CR-113E expansion board: Video Control Module.
+	 -1 x R6520
+	 -1 x HD46505
+	 -2 x EPROM
+	https://www.oldcomputers.es/wp-content/uploads/CR-113.pdf
+	*/
+	PIA6821(config, "pia_cr113e"); // R6520
+	//SCREEN(config, "screen_cr113e", SCREEN_TYPE_RASTER).set_refresh_hz(50).set_screen_update(...);
+	//HD6845S(config, "crtc_cr113e", AIM65_CLOCK).set_screen("screen_cr113e"); // Hitachi HD46505, unknown clock
+
+	/* 
+	CR-119C expansion board: RAM/ROM/PROM Memory Expansion.
+	 -6 x D4016C-2
+	 -1 x 4 dip switches bank
+	 -2 x EPROM
+	https://www.oldcomputers.es/wp-content/uploads/CR-119.pdf
+	*/
+
+	/* 
+	CR-120D.E expansion board: EPROM Programmer Module.
+	 -2 x R6520
+	 -1 x 4 dip switches bank
+	 -1 x EPROM
+	https://www.oldcomputers.es/wp-content/uploads/CR-120.pdf
+	https://www.oldcomputers.es/wp-content/uploads/CR-120-2.pdf
+	*/
+	PIA6821(config, "pia1_cr120d"); // R6520
+	PIA6821(config, "pia2_cr120d"); // R6520
+
+	/* 
+	CR-125 expansion board: Floppy Control Card.
+	 -1 x R6503P ()
+	 -2 x R6522P
+	 -1 x FD1791A
+	 -1 x 4 dip switches bank
+	 -1 x EPROM
+	 -1 x floppy connector
+	https://www.oldcomputers.es/wp-content/uploads/CR-125.pdf
+	*/
+	MOS6522(config, "via1_cr125", AIM65_CLOCK); // R6522P, unknown clock
+	MOS6522(config, "via2_cr125", AIM65_CLOCK); // R6522P, unknown clock
+	FD1791(config, "fdc_cr125", AIM65_CLOCK); // FD1791A, unknown clock
+	//FLOPPY_CONNECTOR(...)
+	M6503(config, "cpu_cr125", AIM65_CLOCK);
```

Comment:
If these are expansions and not fixed parts of the system, they should be in card devices rather than the machine itself.  From what I understand, this system uses a standard AIM 65 logic board, and the expansion box connects to the J3 expansion connector, which is present on all AIM 65 systems, and just not emulated.

From looking at the AIM 65 manual, the expansion connector J3 seems to expose the 6502 bus directly, with the same pinout as the KIM-1 expansion connector B.

From looking at the schematic of the CR 113 board in the manual, it seems to use a different pinout to the J3 connector on the logic board, so I guess the card connectors in the DRAC-1 expansion box have the signals from the J3 expansion connector rearranged.

Reply:
I'll just add then the DRAC-1 as another BIOS, and I'd appreciate if someone else will preserve somehow the other dumps as devices. Take into account that DRAC-1 uses a backplane so multiple expansion cards may be connected simultaneously, don't know if this feature is supported for the AIM 65 devices...
I'll have a look at the DRAC-1 BASIC, as it seems different than the regular Extended BASIC.

Thanks!

Reply:
Can you leave this open as a reminder?  Eventually someone will pick it up, like the Super Game Module from #10816 was done as a proper slot device in ad7fdda12b4cade7032a0e34c3f6ae6dd3964cfa.

Reply:
Yes! Opened a separate PR, #12387